builtin/settingtypes.txt

```diff
@@ -1102,7 +1101,7 @@ mgv6_np_terrain_higher (Terrain higher noise) noise_params_2d 20, 16, (500, 500,
 mgv6_np_steepness (Steepness noise) noise_params_2d 0.85, 0.5, (125, 125, 125), -932, 5, 0.7, 2.0, eased
 
 #    Defines distribution of higher terrain.
-mgv6_np_height_select (Height select noise) noise_params_2d 0.5, 1, (250, 250, 250), 4213, 5, 0.69, 2.0, eased
+mgv6_np_height_select (Height select noise) noise_params_2d 0, 1, (250, 250, 250), 4213, 5, 0.69, 2.0, eased
```

Comment:
This line should not have been changed.  I don't know how this change snuck in, but it could break old v6 worlds (very slightly, but a breakage nonetheless).

Edit: This was my mistake.  That change got into my branch earlier.  Don't know how or why...

Reply:
Aren't these added to `map_meta.txt` either way? I don't see how this could break older worlds (even though this change is obv unnecessary for this PR).

Reply:
Fair point.  Though this change shouldn't have happened in the first place.